.github/workflows/static_page_verification.yml

```diff
@@ -0,0 +1,54 @@
+name: Cloud Formation Template Static Page verification
```

Comment:
Change destination branch from master to dev. Make sure your code works with latest template from dev branch.